### PR TITLE
Added several examples

### PR DESCRIPTION
**Pull request by mikibonacci**
- diamond
- graphite
- GaAs => bands are still a bit not good
- Al

### CODE REVIEW

File: convert-to-json.py [hunk 1]
```diff
@@ -15,22 +15,23 @@
 
 system_name = os.path.basename(os.path.realpath(folder_name))
 
-with open(os.path.join(folder_name, "scf.in")) as fhandle:
+with open(os.path.join(folder_name, "GaAs.scf.in")) as fhandle:
```
Comment (eimrek) [inline convert-to-json.py:18]: I think here we should keep the convention that's defined above, i.e. forcing to use filenames of `scf.in`, `scf.out`, `matdyn.modes`. Would it make sense to rename the files as such? then in principle it's easy to regenerate the json files without having to manually change this code here.

File: convert-to-json.py [hunk 2]
```diff
@@ -15,22 +15,23 @@
...
     scf_input = fhandle.read()
-with open(os.path.join(folder_name, "scf.out")) as fhandle:
+with open(os.path.join(folder_name, "GaAs.scf.out")) as fhandle:
     scf_output = fhandle.read()
-with open(os.path.join(folder_name, "matdyn.modes")) as fhandle:
+with open(os.path.join(folder_name, "GaAs.modes")) as fhandle:
     matdyn_modes = fhandle.read()
 
-pretty_name_dict = {"BaTiO_3": "BaTiO<sub>3</sub>"}
+pretty_name_dict = {"GaAs": "GaAs"}
```
Comment (eimrek) [inline convert-to-json.py:25]: I think this dict should contain entries for all the examples that have a subscript. So GaAs, is not needed, but BaTiO_3 should be kept and everything else that has subscripts should be added (although i'm not sure why it wasn't before). If the entry doesn't exist, i think the rest of the code will just use the correct system name automatically.

File: convert-to-json.py [hunk 3]
```diff
@@ -15,22 +15,23 @@
...
 
-highsym_qpts_default = [[0, "Γ"], [20, "M"], [40, "K"], [60, "Γ"]]
+highsym_qpts_default = [[0, "L"], [40, "Γ"], [80, "K"], [100, "X"], [140, "Γ"]]
```
Comment (eimrek) [inline convert-to-json.py:27]: Here, this is the default qpts, that was the same for all the 2D materials. Before, the only expection was BaTiO3. I think this should be kept as it was, unless there is a better default.

File: convert-to-json.py [hunk 4]
```diff
@@ -15,22 +15,23 @@
...
 highsym_qpts_dict = {
-    "BaTiO_3": [[0, "X"], [20, "Γ"], [40, "M"], [60, "Γ"], [100, "R"]],
+    "GaAs": [[0, "L"], [40, "Γ"], [80, "K"], [100, "X"], [140, "Γ"]],
```
Comment (eimrek) [inline convert-to-json.py:29]: Here you should put all the qpts that do not use the default qpts defined above. I think this is all the 3d examples that you added. BaTiO3 should also be kept here.

File: convert-to-json.py [hunk 5]
```diff
@@ -15,22 +15,23 @@
...
 }
 
-starting_reps_default = (5, 5, 1)
-starting_reps_dict = {"BaTiO_3": (3, 3, 3)}
+#supercell.
+starting_reps_default = (3,3,3)
+starting_reps_dict = {"GaAs": (3, 3, 3)}
```
Comment (eimrek) [inline convert-to-json.py:34]: same here. by default, use (5, 5, 1), but for 3d materials, use (3, 3, 3).

**Review by eimrek — CHANGES_REQUESTED** [review]
thanks @mikibonacci, the tool seems to be working well and the examples are great. Just in the `convert-to-json.py` file, the idea was that it would work without manually changing it for any of the example folders (just by putting the folder path as an argument). Therefore, the code should contain meta information about all the systems (e.g. the qpts to use). it also will act as a documentation on how the final json file was produced in each case and so that they could be easily recreated, if needed (e.g. if there are some updates/corrections to the files). If you can just adapt it accordingly, i'd be happy to approve/merge this.

**Comment by mikibonacci** [comment]
Hi @eimrek, thanks a lot for the review. I will fix what you suggested as soon as possible. Also, I am checking the correctness of the qpath for my new added examples.

File: convert-to-json.py (hunk 1 above)
Comment (mikibonacci) [reply convert-to-json.py:18]: I agree, I was not considering this issue, I'll change it back (and modifiy the filenames accordingly)

File: convert-to-json.py (hunk 2 above)
Comment (mikibonacci) [reply convert-to-json.py:25]: got it

File: convert-to-json.py [hunk 6]
```diff
@@ -19,20 +19,29 @@
     scf_input = fhandle.read()
 with open(os.path.join(folder_name, "scf.out")) as fhandle:
     scf_output = fhandle.read()
-with open(os.path.join(folder_name, "matdyn.modes")) as fhandle:
+with open(os.path.join(folder_name, "modes")) as fhandle:
```
Comment (eimrek) [inline convert-to-json.py:22]: But some files are still using "matdyn.modes" file, e.g. BaTiO_3

**Review by eimrek — CHANGES_REQUESTED** [review]
One small thing to fix & the pre-commit. Will merge when this is done.

**Review by eimrek — APPROVED** [review]
thanks @mikibonacci, everything looks good now :+1:

**Pull request merged**